<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QMainWindow" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="2,10,2" columnstretch="2,10,2">
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" rowspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="fatherwidget" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
           <item>
            <widget class="QWidget" name="boxwidget" native="true">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_7" columnstretch="0,1">
                <item row="0" column="1">
                 <widget class="QLabel" name="boxlabel">
                  <property name="text">
                   <string>Numbers of managers</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="boxIconlabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QLineEdit" name="boxlineEdit">
                  <property name="placeholderText">
                   <string>Enter the number</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QPushButton" name="boxpushButton">
                  <property name="text">
                   <string>Continue</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="formAndSettingwidget" native="true">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0">
                  <item>
                   <widget class="QPushButton" name="soundpushButton">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="darkmodepushButton">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="titleFormlabel">
                    <property name="text">
                     <string>SignUp 1</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="previouspushButton">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="nextpushButton">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QWidget" name="formwidget" native="true">
                  <layout class="QGridLayout" name="gridLayout_4">
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="topMargin">
                    <number>30</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>30</number>
                   </property>
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,1,0,1,0" columnminimumwidth="0,0,0,0,0">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="spacing">
                      <number>10</number>
                     </property>
                     <item row="8" column="2" colspan="2">
                      <widget class="QLineEdit" name="capchalineEdit">
                       <property name="placeholderText">
                        <string>Enter Capcha</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2">
                      <widget class="QLabel" name="passwordIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="6" column="1" colspan="4">
                      <widget class="QLineEdit" name="moneylineEdit">
                       <property name="placeholderText">
                        <string>Bank asset</string>
                       </property>
                      </widget>
                     </item>
                     <item row="6" column="0">
                      <widget class="QLabel" name="bankassetIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="emailIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="11" column="1">
                      <spacer name="verticalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item row="4" column="2" colspan="3">
                      <widget class="QLineEdit" name="phonenumberlineEdit">
                       <property name="placeholderText">
                        <string>Phone Number</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="4">
                      <widget class="QPushButton" name="hidepasswordpushButton">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="3">
                      <widget class="QLineEdit" name="passlineEdit">
                       <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                       </property>
                       <property name="placeholderText">
                        <string>Password</string>
                       </property>
                      </widget>
                     </item>
                     <item row="8" column="0">
                      <widget class="QLabel" name="capchaIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="0">
                      <widget class="QLabel" name="phonenumberIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="1">
                      <widget class="QComboBox" name="selectcountrycomboBox">
                       <property name="styleSheet">
                        <string notr="true"/>
                       </property>
                      </widget>
                     </item>
                     <item row="12" column="0" colspan="5">
                      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2">
                       <item>
                        <spacer name="horizontalSpacer_4">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QPushButton" name="submitAndNextpushButton">
                         <property name="text">
                          <string>submit</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_5">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="usernameIconlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1" colspan="4">
                      <widget class="QLineEdit" name="emaillineEdit">
                       <property name="placeholderText">
                        <string>Email Adress</string>
                       </property>
                      </widget>
                     </item>
                     <item row="8" column="4">
                      <widget class="QPushButton" name="generatecapchapushButton">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="8" column="1">
                      <widget class="QLabel" name="capchaimagelabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="userlineEdit">
                       <property name="placeholderText">
                        <string>Username</string>
                       </property>
                      </widget>
                     </item>
                     <item row="9" column="0" colspan="5">
                      <widget class="QLabel" name="capchaErrorlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="7" column="0" colspan="5">
                      <widget class="QLabel" name="bankassetErrorlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="5" column="0" colspan="5">
                      <widget class="QLabel" name="phonenumberErrorlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0" colspan="5">
                      <widget class="QLabel" name="userpassErrorlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0" colspan="5">
                      <widget class="QLabel" name="emailErrorlabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
